### DISCUSSION TITLE
No module after installing pyvista for python

### DISCUSSION THREAD

**Discussion by prusso630** [pyvista/pyvista#5443] [Q&A]
I tried so far:
pip install pyvista
sudo pip install pyvista

git clone https://github.com/pyvista/pyvista/issues?q=no+module+pyvista
cd ~/pyvista/sudo pip install .
(Said it installed to a package unamed.)

for release:
Distributor ID:	Ubuntu
Description:	Ubuntu 22.04.3 LTS
Release:	22.04
Codename:	jammy

So far in python the pyvista is either not found and code highliting is off or code highlighting is on and I get a no found error. Any way to fix this for ubuntu jammy?

**Comment by banesullivan** [comment]
This sounds like a general Python environment issue. I suspect you're also having trouble installing other libraries (try installing `pandas` for example). I recommend you take a look at tooling like `pyenv` to manage Python virtual environments. https://github.com/pyenv/pyenv
